pkg/host/hostname.go

```diff
@@ -0,0 +1,25 @@
+package host
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+// GetHostname returns the node hostname using kubernetes as lib to ensure that will obtain the same value
```

Comment:
```suggestion
// GetHostname returns the node hostname using kubernetes as lib to ensure that will obtain the same value
// This code is the same used kubernetes project: (We copied to avoid to add a dependency only because of this piece)
//https://github.com/kubernetes/kubernetes/blob/6b34fafdaf5998039c7e01fa33920a641b216d3e/staging/src/k8s.io/component-helpers/node/util/hostname.go#L25-L46
```